services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -471,6 +471,25 @@ async def create_output_dirs(request_mode: CreateDirsRequestItem) -> None:
             dir_to_create.mkdir(parents=True, exist_ok=True)
 
 
+@containers_router.post(
+    "/containers/volumes:permissions",
+    summary="Makes permissions the same on all mounted volumes directories.",
+    response_class=Response,
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def volumes_fix_permissions() -> None:
+    # NOTE: by creating a hidden file on all mounted volumes
+    # the same permissions are ensured and avoids
+    # issues when starting the services
+    mounted_volumes: MountedVolumes = get_mounted_volumes()
+    for volume_path in [
+        mounted_volumes.disk_inputs_path,
+        mounted_volumes.disk_outputs_path,
+    ] + list(mounted_volumes.disk_state_paths()):
+        hidden_file = volume_path / ".hidden_do_not_remove"
+        hidden_file.write_text("directory must not be empty")
```

Comment:
write as well  a reference to __file__ ... can help us debug in the future and realize who and why this was created there

Reply:
```
hidden_file.write_text(f"directory must not be empty. Created by {__file__} ... some useful info of why this is here?")
```